applications/GeoMechanicsApplication/custom_constitutive/small_strain_umat_law.cpp

```diff
@@ -202,13 +203,9 @@ int SmallStrainUMATLaw<TVoigtSize>::Check(const Properties&   rMaterialPropertie
                                           const GeometryType& rElementGeometry,
                                           const ProcessInfo&  rCurrentProcessInfo) const
 {
-    // Verify Properties variables
-    if (!rMaterialProperties.Has(UDSM_NAME) || rMaterialProperties[UDSM_NAME].empty())
-        KRATOS_ERROR << "UDSM_NAME has Key zero, is not defined for property"
-                     << rMaterialProperties.Id() << std::endl;
-
-    KRATOS_ERROR_IF_NOT(rMaterialProperties.Has(IS_FORTRAN_UDSM))
-        << "IS_FORTRAN_UDSM is not defined for property" << rMaterialProperties.Id() << std::endl;
+    const CheckProperties check_properties(rMaterialProperties, "property", CheckProperties::Bounds::AllExclusive);
+    check_properties.CheckAvailabilityAndEmpty(UDSM_NAME);
+    check_properties.CheckAvailabilityOnly(IS_FORTRAN_UDSM);
```

Comment:
When reading the use here, I think CheckAvailability is clear enough:
```suggestion
    check_properties.CheckAvailability(IS_FORTRAN_UDSM);
```